decidim-core/app/assets/stylesheets/decidim/extras/_external-links.scss

```diff
@@ -26,3 +26,21 @@
     height: .9rem;
   }
 }
+
+.external-link-page-content{
+  .external-link{
```

Comment:
After changing the element to the card styling, we can remove the other styles here than the `background-color`.